cli/codeheat.py

```diff
@@ -3,12 +3,14 @@
 from __future__ import print_function, unicode_literals
 from PyInquirer import style_from_dict, Token, prompt, Separator, Validator, ValidationError
 from github import Github
-import github3
+import github3 
 import os, sys, argparse, textwrap, requests, datetime, operator
 
-"""CODEHEAT RUNS FROM SEPTEMBER 10TH, 2018 TO FEBRUARY 1ST, 2019"""
+"""CODEHEAT RUNS FROM SEPTEMBER 15TH, 2019 TO FEBRUARY 2ND, 2020""" #Previous edition RAN from 10th Sept 2018 to 1st Feb 2019
+
 now = datetime.datetime.now()
-CODEHEAT_START, CODEHEAT_END = datetime.datetime(now.year, 9, 10), datetime.datetime(now.year + 1, 2, 1)
+CODEHEAT_START, CODEHEAT_END = datetime.datetime(2019, 9, 15), datetime.datetime(2020, 2, 2) # Previously datetime.datetime(now.year, 9, 10), datetime.datetime(now.year + 1, 2, 1) 
```

Comment:
Don't hard-code the year

Reply:
now.year will cause issues with the end date. In January the contest year originally say 19-20 will become 20-21. I wrote a new algorithm to figure out the contest year.